09_reasoning_about_code/js/src/main/scala/reasonnng_about_the_code/Canvas.scala

```diff
@@ -1,4 +1,4 @@
-package canvas
+package reasonnng_about_the_code
```

Comment:
```suggestion
package reasonnig_about_code
```
(the directory name should be fixed as well)